test/ConnectTest.php

```diff
@@ -64,7 +64,19 @@ public function testUnknownHostConnect()
             $ldap->connect()->bind('CN=ignored,DC=example,DC=com', 'ignored');
             $this->fail('Expected exception for unknown host');
         } catch (Exception\LdapException $zle) {
-            $this->assertContains('Can\'t contact LDAP server', $zle->getMessage());
+            $alternatives = ['Can\'t contact LDAP server',
+                          'Failed to connect to LDAP server'];
```

Comment:
Please reformat this according to [PSR-2](http://www.php-fig.org/psr/psr-2/#4-6-method-and-function-calls) meaning opening and closing parenthesis each on its own line and the two strings in between